src/app/components/search-facet-range/search-range-facet.component.html

```diff
@@ -0,0 +1,22 @@
+<div appCypressData="selected-facets-min">{{selectedRange.start}}</div>
+<div appCypressData="selected-facets-max">{{selectedRange.end}}</div>
+
+<onto-search-facet *ngIf="data"
+                   [data]="data"
+                   [type]="type"
+                   [facetTitleTemplate]="facetTitleTemplate"
+                   [facetTemplate]="basicFacetTemplate"
+                   (onSelectionChange)="onSelectedEvent($event)">
+</onto-search-facet>
+
+<ng-template #facetTitleTemplate>
+  <div class="basic-facet-title">
+    <strong>{{data?.facetGroupName}}</strong>
+  </div>
+</ng-template>
+
+<ng-template #basicFacetTemplate let-item let-sum="sum">
```

Comment:
This context should be documented somewhere

Reply:
done